### PR TITLE
Added get_streams method to get the list of streams in a session

### PR DESCRIPTION
**Pull request by normanargueta**

### CODE REVIEW

File: opentok/opentok.py [hunk 1]
```diff
@@ -535,6 +535,29 @@ def get_stream(self, session_id, stream_id):
         else:
             raise RequestError('An unexpected error occurred', response.status_code)
 
+    def get_streams(self, session_id):
```
Comment (msach22) [inline opentok/opentok.py]: please use `list_streams` as the API

File: opentok/stream.py [hunk 2]
```diff
@@ -1,8 +1,55 @@
...
+        """
+        return dict((k, v) for k, v in iteritems(self.__dict__))
+
+    def json(self):
+        """
+        Returns a JSON representation of the Stream.
+        """
+        return json.dumps(self.attrs(), default=dthandler, indent=4)
+
+class StreamList(object):
```
Comment (msach22) [inline opentok/stream.py]: StreamList should be it's own file so it's easier to maintain and read

**Comment by msach22** [comment]
Please also add info to `README`

**Comment by normanargueta** [comment]
@msach22 I updated the PR with the suggestions you made, could you review it again please? Thanks!

**Pull request merged**